packages/backend/src/connector/core/middleware/account.ts

```diff
@@ -2,23 +2,54 @@ import { Errors } from 'ilp-packet'
 import { RafikiAccount, ILPContext, ILPMiddleware } from '../rafiki'
 import { AuthState } from './auth'
 import { AccountNotFoundError } from '../errors'
+import { OutgoingHttp } from '../services/client'
+import { validateId } from '../../../shared/utils'
 
-export function createAccountMiddleware(): ILPMiddleware {
+export interface OutgoingState {
+  outgoing?: OutgoingHttp
```

Comment:
I'm in the process of also moving `maxPacketAmount` from account to peer, and removing `stream.enabled` (the connector will identify outgoing account as peer and disable stream).
I currently have this as:
```
export interface OutgoingState {
  outgoing?: {
    http?: OutgoingHttp
    stream: {
      enabled: boolean
    }
  }
}
```
But factoring in your other comments, I'm wondering if the connector should just make `RafikiAccount` like I currently have [`MockIlpAccount`](https://github.com/interledger/rafiki/blob/bw-peers/packages/backend/src/connector/core/test/mocks/accounts-service.ts#L6-L18), and hold all of the optional peer fields instead of matching the now reduced accounts model.
So we'd go back to getting this info from `ctx.accounts.outgoing` vs `ctx.state.outgoing`

Reply:
> make RafikiAccount like I currently have MockIlpAccount, and hold all of the optional peer fields instead of matching the now reduced accounts model.
:+1: that sounds cleaner, especially if they were going to get joined into the account query anyway.